packages/universal-components/src/RangeDatePicker/DraggableItem.web.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React from 'react';
```

Comment:
Is this component working for you in this PR? When I try it `DraggableItem.js` is used on web instead of this one. It started working after I've renamed `DraggableItem.js` to `DraggableItem.android.js`. Then proper `DraggableItem.web.js` was used on web.

Reply:
Thanks for the info, that could be possible. I renamed it back  at the end because I wanted to make Android changes in different PR.

Reply:
👍 Now it works fine. Thanks!